docs/docs_source/build_test_ship/validator.rst

```diff
@@ -0,0 +1,86 @@
+
+..  _validator:
+
+WOPI Validation application
+===========================
+
+To assist hosts in verifying their WOPI implementation, Office Online provides a WOPI Validation application that
+executes a test suite against a host's WOPI implementation. The test suite verifies a variety of things, including that
+the semantics for all of the WOPI operations (:ref:`CheckFileInfo`, :ref:`GetFile`, :ref:`PutFile`, etc.) are correct
+and that request/response headers are set properly. New tests are added regularly.
+
+The WOPI Validation application is an Office Online application similar to Word Online or PowerPoint Online.
+It uses the ``.wopitest`` file extension. The WOPI Validation application is included in the :ref:`discovery`
+XML just like all other Office Online applications.
+
+..  code-block:: xml
+
+    <app name="WopiTest" checkLicense="true">
+        <action name="view" urlsrc="https://onenote.officeapps-df.live.com/hosting/WopiTestFrame.aspx" ext="wopitest"/>
+        <action name="getinfo" urlsrc="https://onenote.officeapps-df.live.com/hosting/GetWopiTestInfo.ashx" ext="wopitest"/>
+    </app>
+
+The validation application provides two :ref:`WOPI Actions`, :wopi:action:`view` and :wopi:action:`getinfo`, which
+can be used to trigger the test suite.
+
```

Comment:
Maybe add a comment here saying "We're also planning to open source the WOPI Validator framework soon which might enable other scenarios for hosts, like running pre-checkin tests on a developer's machine."